cached_network_image/pubspec.yaml

```diff
@@ -9,8 +9,11 @@ dependencies:
     sdk: flutter
   flutter_cache_manager: ^3.3.0
   octo_image: ^1.0.0
-  cached_network_image_platform_interface: ^2.0.0
-  cached_network_image_web: ^1.0.2
+  cached_network_image_web:
+    git:
+      url: https://github.com/SuaMusica/flutter_cached_network_image.git
+      ref: feature/errorListener
+      path: cached_network_image_web
```

Comment:
I'm not completely sure, but should this be included in this PR?

Reply:
We need this to use the branch and compile.

When it's merged it should be removed